### ISSUE TITLE
Document explicitly installation of modules as user

### ISSUE THREAD

**Issue by rbrito** [coursera-dl/coursera#320]
To prevent problems like those of #314, we should document (recommend) explicitly the use of installation of modules as user (to not conflict with the ones from the system). Or, better yet, promote the idea of using a venv (is this too technical for users?).

Suggestions are welcome.

**Comment by victorwestmann** [comment]
Yes. I am having some troubles using (making it run, to be hoset) coursera-dl on my Windows 10 pc. And I read the documentation. Twice already. :-)
